app/controllers/bill_controller.rb

```diff
@@ -0,0 +1,26 @@
+class BillController < ApplicationController
+
+  def index
+    @bills = Bill.all
+  end
+
+  def new
+    @bill = Bill.new
+  end
+
+  def create
```

Comment:
The way I envision it , the new route is a GET from the merchant side to a page that allows them to create a bill - after the waiter has physically taken the order from the table. They  assign items from their menu to the bill and hit a POST request that creates the bill. I agree this is where it should pass the json to the client's mobile device.